src/Behavioral.Automation/Services/IElementSelectionService.cs

```diff
@@ -7,5 +8,8 @@ public interface IElementSelectionService
     {
         [CanBeNull]
         IWebElement Find([NotNull] string caption);
+
+        [CanBeNull]
+        IEnumerable<IWebElement> FindMultipleElements([NotNull] string caption);
```

Comment:
Can't be null, please remove the attribute CanBeNull

Reply:
Thanks, fixed